<a href="https://colab.research.google.com/github/pree1207/data-Imputational-Technique/blob/main/mean_median%20imputation.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import numpy as np
from sklearn.impute import SimpleImputer

data = [[80], [85], [90], [95], [np.nan]]
imputer = SimpleImputer(strategy='mean')
imputed_data = imputer.fit_transform(data)
print(imputed_data)


[[80. ]
 [85. ]
 [90. ]
 [95. ]
 [87.5]]


Consider a dataset with student test scores: [80, 85, 90, 95, NaN]. The missing value can be imputed with the mean of the available scores: (80 + 85 + 90 + 95) / 4 = 87.5.